eva-accession-pipeline/src/main/resources/application.properties

```diff
@@ -2,23 +2,28 @@ spring.batch.job.names=
 
 accessioning.instanceId=
 accessioning.submitted.categoryId=ss
-
 accessioning.monotonic.ss.blockSize=100000
 accessioning.monotonic.ss.blockStartValue=5000000000
 accessioning.monotonic.ss.nextBlockInterval=1000000000
 
 parameters.assemblyAccession=
 parameters.taxonomyAccession=
 parameters.projectAccession=
-
 parameters.vcf=
 parameters.vcfAggregation=
 parameters.fasta=
-
+parameters.assemblyReportUrl=
+parameters.outputVcf=
 parameters.chunkSize=
 parameters.forceRestart=
+parameters.contigNaming=SEQUENCE_NAME
 
 spring.data.mongodb.database=
+spring.data.mongodb.host=
+spring.data.mongodb.port=
+spring.data.mongodb.username=
+spring.data.mongodb.password=
+spring.data.mongodb.authentication-database=
```

Comment:
When I tried running the JAR built from this fork, I faced some issues running the pipeline against a passwordless accessioning database in the Embassy Cloud because the program used the blank username and password from this file. Can the username, password and authentication-database parameters be taken out?